rhmap-test-driver/app/index.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const Utils = require("./src/utils");
+const TestRunner = require("./src/test-runner");
+
+const endPoint = process.argv[2];
+const appId = process.argv[3];
+const csvPath = process.argv[4];
+const rawDelay = process.argv[5];
+
+if (!endPoint || !appId || !csvPath || !rawDelay) {
```

Comment:
I would suggest to use a command line parser so that we could use a command line like:
`--endPoint <endpoint> --appId <appId> --csv <path> --delay <delay>`

As it is now, I could erroneously switch the parameters and cal:
`app appId rawDelay csvPath endPoint`

An example could be https://github.com/tj/commander.js/